src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -392,32 +407,25 @@ private void completeInner(FlightContext flightContext) throws SQLException {
             + " stairway_id = NULL"
             + " WHERE flightid = :flightId AND status = 'RUNNING'";
 
-    // The delete is harmless if it has been done before. We just won't find anything.
-    final String sqlDeleteFlightLog =
-        "DELETE FROM " + FLIGHT_LOG_TABLE + " WHERE flightid = :flightId";
-
     String serializedException =
         exceptionSerializer.serialize(flightContext.getResult().getException().orElse(null));
 
     try (Connection connection = dataSource.getConnection();
         NamedParameterPreparedStatement statement =
-            new NamedParameterPreparedStatement(connection, sqlUpdateFlight);
-        NamedParameterPreparedStatement deleteStatement =
-            new NamedParameterPreparedStatement(connection, sqlDeleteFlightLog)) {
+            new NamedParameterPreparedStatement(connection, sqlUpdateFlight)) {
 
       startTransaction(connection);
 
+      // TODO(PF-703): Column output_parameters is being phased out in favor of table
+      //               flightworking.  We continue to write the output map here
+      //               temporarily for backward compatibility.
       statement.setString("outputParameters", flightContext.getWorkingMap().toJson());
+
       statement.setString("status", flightContext.getFlightStatus().name());
       statement.setString("serializedException", serializedException);
       statement.setString("flightId", flightContext.getFlightId());
       statement.getPreparedStatement().executeUpdate();
 
```

Comment:
Per our discussion, `keepFlightLog` is no longer obeyed, and logs are always kept (this avoids FK violations and allows us to use last `flightworking` as `output_parameters`.

Reply:
worth an update to the PR description